terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 1: 23551 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 2: 23553 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 3: 23555 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 4: 23557 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 5: 23559 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 6: 23561 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 7: 23563 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 8: 23565 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 9: 23567 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 10: 23569 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 11: 23571 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 12: 23573 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 13: 23575 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 14: 23577 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 15: 23579 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 16: 23581 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 17: 23583 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 18: 23585 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 19: 23587 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 20: 23589 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 21: 23591 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 22: 23593 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 23: 23595 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 24: 23597 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 25: 23599 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 26: 23601 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 27: 23603 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 28: 23605 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 29: 23607 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 30: 23609 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 31: 23611 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 32: 23613 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 33: 23615 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 34: 23617 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 35: 23619 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 36: 23621 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 37: 23623 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 38: 23625 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 39: 23627 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 40: 23629 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 41: 23631 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 42: 23633 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 43: 23635 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 44: 23637 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 45: 23639 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 46: 23641 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 47: 23643 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 48: 23645 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 49: 23647 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 50: 23649 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 51: 23651 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 52: 23653 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 53: 23655 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 54: 23657 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 55: 23659 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 56: 23661 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 57: 23663 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 58: 23665 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 59: 23667 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 60: 23669 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 61: 23671 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 62: 23673 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 63: 23675 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 64: 23677 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 65: 23679 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 66: 23681 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 67: 23683 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 68: 23685 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 69: 23687 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 70: 23689 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 71: 23691 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 72: 23693 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 73: 23695 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 74: 23697 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 75: 23699 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 76: 23701 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 77: 23703 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 78: 23705 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 79: 23707 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 80: 23709 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 81: 23711 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 82: 23713 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 83: 23715 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 84: 23717 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 85: 23719 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 86: 23721 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 87: 23723 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-c.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 88: 23725 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 89: 23727 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-d.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 90: 23729 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 91: 23731 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 92: 23733 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 93: 23735 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 94: 23737 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 95: 23739 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 96: 23741 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 97: 23743 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-a.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 98: 23745 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-b.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 99: 23747 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 100: 23749 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/14.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-3-1-d-val.sh: line 101: 23751 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-d.pddl $1
